ruby/lib/grpclb/internal/load_meter.rb

```diff
@@ -0,0 +1,40 @@
+# TODO: would be nice to have some tests for it
+
+module Grpclb
+  module Internal
+    class LoadMeter
+      def initialize(period_seconds: 60)
+        @period = period_seconds
+
+        @mutex = Mutex.new
+        @scored_at = Time.now
+        @count = 0
+      end
+
+      def increment(inc=1)
+        @mutex.synchronize do
+          @count += inc
+        end
+      end
+
+      def score
+        @mutex.synchronize do
+          now = Time.now
+          passed = now - @scored_at
+          @scored_at = now
+
+          break 0 if passed == 0
+
+          if passed < @period
+            @scored_at -= passed
+            break @count * @period / passed
```

Comment:
sorry @mxmCherry, but I have not idea what is going on here (and in the lines above and below), can you please make revise and untangle

Reply:
That's roughly a port of [load.RateReporter](https://github.com/bsm/grpclb/blob/master/load/reporter.go#L54)

Reply:
Initially, I wanted to use https://github.com/johnewart/ruby-metrics/blob/master/lib/ruby-metrics/instruments/meter.rb , but decided not to introduce this dep just for this single class.

I can switch to it, as, unfortunately, we cannot get `GRPC::Server` pool.